lib/alephant/logger/cloudwatch.rb

```diff
@@ -3,8 +3,11 @@
 module Alephant
   module Logger
     class CloudWatch
-      def initialize(namespace)
-        @namespace = namespace
+      def initialize(namespace, unit = "Count", value = 1, dimensions = {})
```

Comment:
Let's use an options hash here so we can avoid using multiple optional positional arguments